Van_Marukyan/python/pascal_triangle/pascal.py

```diff
@@ -0,0 +1,10 @@
+def pascal_triangle(n):
+    print("\n", "pascals triangle for" , n , "\n")
+    trow = [1]
+    y = [0]
+    for x in range(max(0,n)):
+        print(" " , trow)
+        trow=[l+r for l,r in zip(trow+y, y+trow)]
+    return n>=1
+
+pascal_triangle(10) 
```

Comment:
- For 0 and 1 the vales are  incorrect
![image](https://user-images.githubusercontent.com/80816144/119727199-10654e80-be83-11eb-8135-156b2168675c.png)
- No any user input
- No any input validation
![image](https://user-images.githubusercontent.com/80816144/119727252-207d2e00-be83-11eb-9a01-f1eb4da19136.png)